README.md

```diff
@@ -435,7 +435,7 @@ action.call({id: 'unknown'}) # => raises RecordNotFound
 #### Inherited Exceptions
 
 ```ruby
-class MyCustomException < StandardError
```

Comment:
@karimmtarek This is unrelated to the scope of your change, please revert it. :smile: